### PR TITLE
Big readme update, includes OpenBW on the front page

### PR DESCRIPTION
**Pull request by ebetica**
I built it the OpenBW instructions and ran it to the point where I got the TorchCraft config displaying on an empty docker instance, so it builds fairly easily.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,64 +9,75 @@ Richoux, F. and Usunier, N., 2016. _TorchCraft: a Library for Machine Learning R
...
 You will need to have a license of StarCraft: Brood War. 
 
-The most tested and stable way to set it up is to have StarCraft running on Windows 
-and Torch running in Linux or Mac OS X. If you already have StarCraft, you need to
-install [BWAPI](docs/user/installation.md#bwapi),
-[copy a few files](docs/user/installation.md#torchcraft-common-prerequisites), 
-and either launch the [AIModule (DLL)](docs/user/installation.md#torchcraft-aimodule-dll-for-users) 
-OR launch the [AIClient (EXE)](docs/user/installation.md#torchcraft-aiclient-exe-for-users). 
+TorchCraft is proxy into BWAPI, that enables data from StarCraft to be sent 
+over a ZMQ connection so that it can be parsed anywhere. The TorchCraft client
```
Comment (dgant) [inline README.md]: Proposed: "TorchCraft is a BWAPI module that sends StarCraft data out over a ZMQ connection. This lets you parse StarCraft data and interact with BWAPI from anywhere."

File: docs/user/starcraft_in_windows.md [hunk 2]
```diff
@@ -32,8 +32,8 @@ AIClient or AIModule.
 - Download the game at https://battle.net/account/management/.
 - Run `setup.exe` and install the game in some folder.
   - `$STARCRAFT` will be from now on the installation directory.
-- Download the 1.16.1 patch at https://battle.net/support/en/article/classic-game-patches.
-  - Make sure you download the **Brood War** patch.
+- Download the 1.16.1 patch at http://ftp.blizzard.com/pub/broodwar/patches/PC/BW-1161.exe
+  - Make sure you download the **Brood War** patch 1.16.1.
```
Comment (dgant) [inline docs/user/starcraft_in_windows.md]: When Blizzard released 1.18 and announced Remastered (1.20) they removed the 1.15 installer that was available on their website. And the 1.16.1 patch doesn't work on either of the newer versions. So I think these instructions, sadly, no longer work.

If I recall correctly, though, the 1.18 MPQs are sufficient for OpenBW (even if the 1.18 executable doesn't itself support BWAPI). Does that sound correct @tscmoo ? If so, we could correct and simplify these instructions to, approximately, "go download 1.18"

Comment (edran) [reply docs/user/starcraft_in_windows.md]: There's also the iccup version which works fine, so we could point to that.

Comment (dgant) [reply docs/user/starcraft_in_windows.md]: @ebetica @tscmoo confirms that the 1.18 MPQs should be sufficient for OpenBW. So we can just tell people to download that from Blizzard and skip the patching, albeit with the disclaimer that they can't use 1.18 directly like with ChaosLauncher.

Comment (ebetica) [reply docs/user/starcraft_in_windows.md]: Linked to a Blizzard forum post on how to downgrade to 1.16.1

**Pull request merged**